src/rsa.rs

```diff
@@ -86,9 +92,29 @@ impl signature_impl::VerificationAlgorithmImpl for RSA_PKCS1 {
     }
 }
 
+macro_rules! rsa_pkcs1_padding {
+    ( $PADDING_ALGORITHM:ident, $digest_alg_name:expr,
+      $digest_alg:expr, $digestinfo_prefix:expr ) => {
+
+        #[doc="PKCS#1 1.5 padding with the "]
+        #[doc=$digest_alg_name]
+        #[doc=" digest algorithm."]
```

Comment:
Are you sure about the first part? Seems to me like nothing about the PKCS1 padding, per se, requires the heap...